packaging/installer/kickstart.sh

```diff
@@ -1360,7 +1360,7 @@ cleanup_apt_cache() {
     cache_dir="/var/cache/apt/archives"
 
     if [ -d "${cache_dir}" ]; then
-        find "${cache_dir}" -t file -name 'netdata*.deb' -delete
+        run_as_root find "${cache_dir}" -t file -name 'netdata*.deb' -delete
```

Comment:
```suggestion
        run_as_root find "${cache_dir}" -type f -name 'netdata*.deb' -delete
```

At least on Debian it is `-type` and the type is a single letter


```
       -type c
              File is of type c:

              b      block (buffered) special

              c      character (unbuffered) special

              d      directory

              p      named pipe (FIFO)

              f      regular file

              l      symbolic  link; this is never true if the -L option or the -follow option is in effect, unless the symbolic link is broken.  If you
                     want to search for symbolic links when -L is in effect, use -xtype.

              s      socket

              D      door (Solaris)

              To search for more than one type at once, you can supply the combined list of type letters separated by a comma `,' (GNU extension)
```